Source/JavaScriptCore/bytecode/StructureStubInfo.cpp

```diff
@@ -536,6 +541,7 @@ void StructureStubInfo::initializeFromUnlinkedStructureStubInfo(const BaselineUn
         break;
     case AccessType::GetByVal:
     case AccessType::GetPrivateName:
+    case AccessType::GetPrivateNameById:
         hasConstantIdentifier = false;
```

Comment:
This is not correct since it has `hasConstantIdentifier`. Let's see how these values are used in `GetPrivateNameById` case and ensure that these configurations are correct.